pcsx2-qt/MainWindow.cpp

```diff
@@ -2905,6 +2910,35 @@ void MainWindow::goToWikiPage(const GameList::Entry* entry)
 	QtUtils::OpenURL(this, fmt::format("https://wiki.pcsx2.net/{}", entry->serial).c_str());
 }
 
+void MainWindow::openScreenshotsFolderForGame(const GameList::Entry* entry)
+{
+    std::string base_path = EmuFolders::Snapshots;
```

Comment:
why are you making a copy? you can pass `EmuFolders::Snapshots` directly to `Path::Combine`